src/hubverse_transform/model_output.py

```diff
@@ -200,14 +200,21 @@ def read_file(self) -> pa.table:
             model_output_file = self.fs_input.open_input_stream(self.input_file)
             # normalize incoming missing data values to null, regardless of data type
             options = csv.ConvertOptions(
-                null_values=["na", "NA", "", " ", "null", "Null", "NaN", "nan"], strings_can_be_null=True
+                null_values=["na", "NA", "", " ", "null", "Null", "NaN", "nan"],
+                strings_can_be_null=True,
+                # temp fix: force location and output_type_id columns to string
+                column_types={"location": pa.string(), "output_type_id": pa.string()},
             )
             model_output_table = csv.read_csv(model_output_file, convert_options=options)
         else:
-            # parquet requires random access reading (because metadata),
-            # so we use open_input_file instead of open_intput_stream
+            # temp fix: force location and output_type_id columns to string
+            schema_new = pq.read_schema(self.input_file)
```

Comment:
@matthewcornell This is the change to ensure that we apply the schema when reading parquet files (rather than doing it after we read into a pyarrow table):

1. Get an arrow schema from the parquet metadata of the original file (so we have all of the column names)
2. Create a new arrow schema, overriding the datatype for location and output_type_id
3. Use the new schema when reading the original file into an pyarrow table